### ISSUE TITLE
Paragraph indent affects boxes starting a paragraph

### ISSUE THREAD

**Issue by Omikhleia** [Omikhleia/ptable.sile#8]
From @nphyx in #7

> When paragraph indentation is enabled, boxes at the start of a paragraph are indented by `document.parindent` from frame left. 
>
> Calling `\noindent` before `\framebox` doesn't solve this.

My answer:

The problem here is that you are hitting a bug (or at least a very weird behavior) of SILE itself, with respect to the measured width of horizontal boxes at the beginning of a paragraph. You would have the same issue with, say, the standard **rules** packages and its `\underline` command (or the deprecated `\boxaround` as well); or the **rotate** package, etc.

I was pretty sure I already had seen that kind of problem discussed on the SILE issue tracker, but just looking (fairly) quickly, I could not find it back. I would suggest to report to SILE, therefore. For what matters, here is an example showing that same issue with standard packages:

```
\begin{document}
\use[module=packages.rules]
\use[module=packages.rotate]

\noindent{}Lorem...

Lorem (paragraph indented)

\underline{ipsum}

\rotate[angle=5]{dolor}

\boxaround{toto}% deprecated command, but just to show it there too

\end{document}
```

![image](https://user-images.githubusercontent.com/18075640/220814501-20043ac2-94b4-459d-a8f3-adba399ad607.png)

_Originally posted by @Omikhleia in https://github.com/Omikhleia/ptable.sile/issues/7#issuecomment-1441163570_

**Comment by Omikhleia** [comment]
Reported to SILE https://github.com/sile-typesetter/sile/issues/1718

**Comment by Omikhleia** [comment]
The related SILE issue was fixed in release v0.14.9. Yay!
